sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyOperationParameters.java

```diff
@@ -145,7 +172,12 @@ public KeyOperationParameters setAdditionalAuthenticatedData(byte[] additionalAu
      * @return The updated {@link KeyOperationParameters} object.
      */
     public KeyOperationParameters setAuthenticationTag(byte[] authenticationTag) {
-        this.authenticationTag = authenticationTag;
+        if (authenticationTag == null) {
```

Comment:
This block can be:
this.authenticationTag = authenticationTag != null ? Base64Url.encode(authenticationTag) : null;